### DISCUSSION TITLE
I have solution to fix iOS animation not run in MAUI.  it's ok for me

### DISCUSSION THREAD

**Discussion by EarnSaranya** [mono/SkiaSharp.Extended#176] [Show and tell]
when I use this code in .xaml 
       <skia:SKLottieView
                    IsAnimationEnabled="true"
          RepeatCount="10"
          RepeatMode="Reverse"
          Source="dotnetbot.json" 
          HeightRequest="400"
          WidthRequest="400" />       
          
          animation not run in IOS 

but I try this 
       <skia:SKLottieView
                    IsAnimationEnabled="{Binding SmartAnimationEnable}"

          RepeatCount="10"
          RepeatMode="Reverse"
          Source="dotnetbot.json" 
          HeightRequest="400"
          WidthRequest="400" />       

and in viewModel. I set SmartAnimationEnable = false; 
in Controller OnAppearing I set SmartAnimationEnable = true ;    
it's RUN  ^_^. 
maybe this solution will can help someone 
Thank you.

**Comment by krirkGenerwiz0137** [comment]
Yes, This Solution is Good for me ^_^
